tests/pages/base_page.py

```
@@ -158,27 +154,68 @@ def click_on_add_item(self):
+            EC.element_to_be_clickable(input_locator)
+        )
+
+        # scroll + click (dropdown aç)
+        self.driver.execute_script(
+            "arguments[0].scrollIntoView({block:'center'});",
+            input_el
+        )
+
+        input_el.click()
+
+        # 🔥 MUI dropdown açılması için küçük bekleme
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF003)